app/controllers/application_controller.rb

```diff
@@ -10,6 +10,12 @@ class ApplicationController < ActionController::Base
   include Chat
   include Localisation
 
+  before_action :fetch_footer_content
+  def fetch_footer_content
+    interactor = Core::FooterReader.new('footer')
```

Comment:
Deep question! It's usually done for readability. Fowler calls them "explaining variables" (see Refactoring, p124 "Introduce Explaining Variable").